pkg/detectors/flexport/flexport.go

```diff
@@ -0,0 +1,100 @@
+package flexport
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"flexport"}) + `\b(shltm_[0-9a-zA-Z-_]{40})\b`)
```

Comment:
Two thoughts:

1. The `shltm_` prefix seems unique enough that the flexport prefix isn't warranted.
2. The pattern `[0-9a-zA-Z-_]{40})\b` wouldn't match things like `abcdef-` (dash at end). Is it possible for tokens like that to be generated?

Reply:
(1) agreed.
(2) that's a really good point. i'm not sure but will check

Reply:
(2) we have not been able to reproduce a token with a dash at the end, but we did find tokens with multiple dashes, and as such i think we should ere on the side of this being possible. as such i think we can remove the ending boundary condition. thanks for pointing it out. does that seem okay?